lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -1503,7 +1519,8 @@ def test_with_override_with_history(self, login_method):
         resp = self.client.get(
             self.get_url(subsection_id=self.usage_key)
         )
-
+        created_date = resp.data['history'][0]['created']
```

Comment:
The `freeze_time` decorator should make it so that both of these are equal to 2019-01-01T00:00.